grouper/fe/util.py

```diff
@@ -305,16 +314,18 @@ def ensure_audit_security(perm_arg):
     """
 
     def _wrapper(f):
+        # type: (Callable[..., None]) -> Callable[..., None]
         def _decorator(self, *args, **kwargs):
+            # type: (GrouperHandler, *Any, **Any) -> None
             if not any(
                 [
                     name == AUDIT_SECURITY and argument == perm_arg
                     for name, argument, _, _ in user_permissions(self.session, self.current_user)
                 ]
             ):
-                return self.forbidden()
```

Comment:
Does `forbidden` stop execution? Haven't dug into it, just noticed that every other callsite seems to follow the `return self.forbidden()` pattern

Reply:
Oh, whoops, yes, this is a bug. Thank you for the catch. This was left over from something else I reverted.